src/PublicApiAnalyzers/Setup/source.extension.vsixmanifest

```diff
@@ -7,7 +7,7 @@
     <Description xml:space="preserve">Public API Analyzers</Description>
   </Metadata>
   <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.0.0.0,17.0.0.0)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0.27130.0,]" />
```

Comment:
Same here, all the changes to VSIX manifests should be persisted.

Reply:
I'm leaving this as a follow-up per https://github.com/dotnet/roslyn-analyzers/pull/2367#discussion_r276790377.